internal/version/version.go

```diff
@@ -6,4 +6,4 @@
 package version
 
 // Tag specifies the current release tag. It needs to be manually updated.
-const Tag = "v0.7.2"
+const Tag = "v0.7.3"
```

Comment:
This is a minor release imo 

```suggestion
const Tag = "v0.8.0"
```